_includes/manuals/1.19/3.1.3_puppet_versions.md

```diff
@@ -73,7 +66,7 @@ Puppet Server is the application for serving Puppet agents used by default in Pu
   <tr>
     <td>2.0 - 2.1</td>
     <td>Limited support</td>
-    <td>Requires server_use_legacy_use_conf, server_puppetserver_version to be set</td>
+    <td>Requires server_use_legacy_host_conf, server_puppetserver_version to be set</td>
   </tr>
   <tr>
     <td>2.2 - 2.7</td>
```

Comment:
2.8 is out and should also work. Maybe we should mark < 2.8 as Untested and 2.8 as Supported. @mmoll?

Reply:
sounds legit :+1: